pallets/automation-time/src/tests.rs

```diff
@@ -785,6 +785,49 @@ fn on_init_shutdown() {
 	})
 }
 
+#[test]
+fn migration_v1() {
+	new_test_ext(START_BLOCK_TIME).execute_with(|| {
+		AutomationTime::on_initialize(1);
+		let new_timestamp = START_BLOCK_TIME + (3600 * 1_000);
+		Timestamp::set_timestamp(new_timestamp);
+		v1::migrate::<Test>();
+		if let Some((updated_last_time_slot, updated_last_missed_slot)) =
+			AutomationTime::get_last_slot()
+		{
+			assert_eq!(updated_last_time_slot, LAST_BLOCK_TIME,);
+			assert_eq!(updated_last_missed_slot, LAST_BLOCK_TIME,);
+		}
+	})
+}
+
+#[test]
+fn migration_v2() {
+	new_test_ext(START_BLOCK_TIME).execute_with(|| {
+		add_task_to_task_queue(ALICE, vec![60], Action::Notify { message: vec![50] });
+		add_task_to_missed_queue(ALICE, vec![60], Action::Notify { message: vec![50] });
+		schedule_task(ALICE, vec![40], SCHEDULED_TIME, vec![2, 4, 5]);
+		LastTimeSlot::<Test>::put((LAST_BLOCK_TIME, LAST_BLOCK_TIME));
+
+		v2::migrate::<Test>();
+		Timestamp::set_timestamp(START_BLOCK_TIME + (3600 * 1_000));
+		if let Some((updated_last_time_slot, updated_last_missed_slot)) =
+			AutomationTime::get_last_slot()
+		{
+			assert_eq!(updated_last_time_slot, LAST_BLOCK_TIME,);
+			assert_eq!(updated_last_missed_slot, LAST_BLOCK_TIME,);
+		}
+		match AutomationTime::get_scheduled_tasks(SCHEDULED_TIME) {
```

Comment:
also need to check the Tasks map

Reply:
gahh i knew i forgot something on this test